utils/index.js

```diff
@@ -7,4 +8,15 @@ const getLabel = (fieldKey, fieldValue) => {
   return translations[fieldKey]?.options[fieldValue]?.label || undefined;
 };
 
-module.exports = { getLabel };
+const formatDate = date => {
+  const dateObj = new Date(date);
+  return new Intl.DateTimeFormat(config.dateLocales).format(dateObj);
```

Comment:
Updated to use specific format. This has been updated in `config.dateFormat`.

The format requested by UCD is the same as default, but now this is explicit in case the default changes.